bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -276,7 +275,13 @@ public Response doHead( final String packageType, final String type, final Strin
                     logger.debug( "Building 200 response. Using HTTP metadata: {}", httpMetadata );
 
                     final ResponseBuilder builder = Response.ok();
-                    setInfoHeaders( builder, item, sk, path, true, contentController.getContentType( path ),
+
+                    // restrict the npm header contentType with json to avoid some parsing error
+                    String contentType = packageType.equals( NPM_PKG_KEY ) ?
```

Comment:
Does this mean that Indy will return application/json content type even for tarballs that are retrieved via a NPM Indy repository?